website/src/data/users.tsx

```diff
@@ -181,7 +181,34 @@ const Users: User[] = [
     source: 'https://github.com/Azure-Samples/todo-nodejs-mongo-aks',
     tags: ['featured','bicep','nodejs','typescript','javascript','mongodb','monitor','keyvault', 'reactjs', 'appservice','cosmosdb','aks'],
   },
-  
+  {
+    title: 'ChatGPT Plugin using Python (FastAPI)',
+    description: 'This is a quickstart for creating a ChatGPT Plugin using Python and Azure.',
+    preview: require('./images/chatgpt-plugin-python.png'),
+    website:'https://github.com/Azure-Samples',
+    author:'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/openai-plugin-fastapi',
+    tags: ['featured', 'bicep','python','aca','cosmosdb','monitor', 'fastapi', 'ai'],
+  }, 
+  {
+    title: 'ChatGPT + Enterprise data with Azure OpenAI and Cognitive Search',
+    description: 'A sample app for the Retrieval-Augmented Generation pattern running in Azure, using Azure Cognitive Search for retrieval and Azure OpenAI large language models to power ChatGPT-style and Q&A experiences.',
+    preview: require('./images/chatgpt-enterprise-dotnet.png'),
+    website:'https://github.com/Azure-Samples',
+    author:'Azure Content Team',
+    source: 'https://github.com/Azure-Samples/azure-search-openai-demo-csharp/',
+    tags: ['featured', 'bicep','dotnetCsharp','aks', 'kubernetes','aca','cosmosdb','cognitivesearch', 'openai', 'chatgpt', 'rediscache', 'ai'],
+  },
+  {
+    title: 'BYO-Chat Application using Azure OpenAI',
+    description: 'This repository shows you the basic code of how to use the OpenAI SDK for Python to build your own chat application with Azure OpenAI',
```

Comment:
It'll be an actual chat (see https://chatfrance-zypx32dl-ca.livelyrock-dc5184bd.francecentral.azurecontainerapps.io/ ) so maybe

```suggestion
    description: 'A sample chat app that uses Python and the Flask framework to stream ChatGPT completions as server-sent events to a simple frontend.',
```

Reply:
Sounds good! I think I just copied this from the repo description so we probably want to update it there as well!

Reply:
Yep, makes sense, I'll update the README in my pending PR